tests/CMakeLists.txt

```diff
@@ -1,19 +1,31 @@
+set(tests_dir ${CMAKE_CURRENT_SOURCE_DIR})
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
-if(SYCL_CTS_TEST_FILTER)
```

Comment:
Why these changes? They seem to go the other direction of what @psalz 's PRs did.

Reply:
Whoops, these changes are from the older code base
I've reverted it in this commit: f18f47ca70a3ec5125fa8fb0046431d373de898f